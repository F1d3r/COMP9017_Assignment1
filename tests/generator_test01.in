-N=1423 -mindist=0.1 -rseed=32