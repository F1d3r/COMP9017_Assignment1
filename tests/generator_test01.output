22.81, 16.60
38.69, 8.34
-5.57, 36.60
39.77, 4.56
-44.44, 30.79
-31.40, -37.52
36.52, -1.29
26.20, 25.81
37.74, 13.64
-13.22, -9.33
2.59, 16.31
-47.27, -48.74
-2.87, 3.91
-41.24, 4.64
-18.49, 48.23
11.18, 1.76
-32.41, 21.36
-15.39, -34.63
44.62, -44.74
-37.72, -17.68
20.63, 12.29
-1.29, 17.78
-28.85, -2.35
3.50, -8.18
8.76, 2.23
