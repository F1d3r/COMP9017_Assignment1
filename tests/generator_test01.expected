23.06, 41.87
41.86, 17.96
-42.16, -47.52
19.93, 30.49
-34.88, 42.46
26.38, -49.66
23.64, 45.62
32.36, -31.13
-17.39, 25.15
-21.96, -26.99
30.76, 25.92
28.14, 39.31
2.40, -44.86
-15.32, -1.98
40.66, 7.05
5.77, 7.18
39.76, -28.55
-17.67, -35.19
18.08, 43.64
6.18, -20.77
